addons/survey/views/survey_templates.xml

```diff
@@ -281,27 +305,56 @@
         <div class="row o_survey_form_choice"
              t-att-data-name="question.id"
              data-question-type="simple_choice_radio">
-            <div t-foreach='question.suggested_answer_ids' t-as='label' t-attf-class="col-lg-#{question.column_nb}">
-                <label t-att-class="' bg-success ' if quizz_correction and label.answer_score > 0.0 else None">
-                    <input type="radio" t-att-value='label.id' class="o_survey_form_choice_item"
-                           t-att-name='question.id'
-                           t-att-checked="answer_line and answer_line.suggested_answer_id.id == label.id and 'checked' or None"/>
-                    <span t-field='label.value'/>
-                </label>
+            <t t-set="letters" t-value="'ABCDEFGHIJKLMNOPQRSTUVWXY'"/>
```

Comment:
Same variables defined several times, should be done once and re-used when necessary :) .